### PR TITLE
Sprint3-1-SearchPets-KW

### PR DESCRIPTION
**Pull request by wessstt**
- Fixed broken image on Slider. 
- Revised Fav section link in Navbar. 
- Made shared component folder. Moved pet data into its own file - _will be API data once we connect._ 
- Moved pet card into its own shared component. 
- Revised favorites state to uncheck icon on search page. 
- Moved Hereo into main components folder. 
- Created PetSearch page with search functionality and filters. - _Still want to add Sort functionality and pagination._ 
- Added routes in App.jsx.

_**I have a console error regarding Dom nesting for MUI and I haven't been able to source that issue yet. 😵‍💫_

### CODE REVIEW

File: src/components/SearchPets.tsx
Comment (Leonardo-Reyes-Munoz) [inline src/components/SearchPets.tsx:1]: Your applyFilters() function looks intense. That must have taken a long time. Nice job Keri!

**Review by Leonardo-Reyes-Munoz — APPROVED** [review]
Looks good Keri! Thank you. That was a lot of work.  I really like the look of the search page and the logic looked like it took a lot of work

**Review by Count-MonteCristo — APPROVED** [review]

**Review by rebs1203 — APPROVED** [review]
Looking very good, Keri! Great job! I'm approving your changes

File: src/components/SearchPets.tsx
Comment (wessstt) [reply src/components/SearchPets.tsx:1]: Thanks, it was quite a bit to wrap my head around haha. 😵‍💫😂 Still needs some fixing but it has the main functionality at least 🙌

**Pull request merged**